daemon/logger/jsonfilelog/read.go

```diff
@@ -305,20 +308,13 @@ func followLogs(f *os.File, logWatcher *logger.LogWatcher, notifyRotate chan int
 		if !since.IsZero() && msg.Timestamp.Before(since) {
 			continue
 		}
+		if !until.IsZero() && msg.Timestamp.After(until) {
+			continue
+		}
 		select {
 		case logWatcher.Msg <- msg:
 		case <-ctx.Done():
 			logWatcher.Msg <- msg
-			for {
```

Comment:
IIRC, this is used when the logger is closed (ie, container stopped) but the consumer is still waiting for logs. It basically reads the rest of the logs until EOF and then returns.

I believe we still need this, perhaps with a comment (or abstracted better?)